buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -530,6 +536,40 @@ private void copyExtraConfigFiles() {
         });
     }
 
+    /**
+     * Copies extra jars to the `/lib` directory.
+     * //TODO: Remove this when system modules are available
+     */
+    private void copyExtraJars() {
+        Configuration extraJarsConfig = project.getConfigurations().findByName("extraJars");
+        if ( extraJarsConfig != null ) {
+            Set<File> extraJars = project.getConfigurations().getByName("extraJars").getFiles();
+            for (File jar : extraJars) {
+                Path destination = getDistroDir().resolve("lib");
+                LOGGER.info("Adding extra jar {} to {}", jar.getName(), destination);
+                project.copy(spec -> {
+                    spec.from(jar);
+                    spec.into(destination);
+                });
+            }
+        }
+    }
+
+    private void configureNodeForFips() {
```

Comment:
We should do this externally, you can use `testclusters.all {}` to configure all clusters in a project, so combining that with `allprojects {}` would allow you to configure it across the board

Reply:
I can't figure out how I can reference the file locations from within the `BuildPlugin` if I do this in testclusters.all {}. Any ideas ?

Reply:
Locations of _what_ files exactly?

Reply:
The fips_java.security, fips_java.policy and cacerts.bcfks that I copy as extra config files

Reply:
That's problematic indeed. `getConfigDir` is public, but we don't have a way to iterate over the nodes, or configure nodes individually.

Does this needs to be an absolute path ? Would a path relative to cwd work ? e.x. `conf/fips_java.policy` i think that's how this problem is solved elsewhere.

Reply:
These paths are passed as values to system properties, not used in ES configuration so we either need an absolute path. Maybe we have a global reference to the ES_CONF_DIR that can be resolved in a system property? I'll look into it

Reply:
Solution required adding `ES_PATH_CONF` to the substitutions we do when parsing `jvm.properties` . Can't think of a side-effect but I wanted to raise it for visibility when this is reviewed